docs/MANAGE.md

```diff
@@ -39,6 +39,13 @@ Services contain basic information such as a name, description, and category, as
 
 ### How to change your data
 
+#### Set up a Firebase user
+
+1. Navigate to the Authentication page of your [Firebase Console](https://firebase.google.com/console).
+2. Make sure you have Email/Password authentication enabled.  It should prompt you to enable it if you hover over *ADD USER*, otherwise you can find the setting under the *SIGN-IN METHOD* tab near the top of the page.
```

Comment:
you can make these all `1.` and they will get evaluated when rendered, makes changing instructions nicer.